lib/cc/analyzer/issue_location_existence_validation.rb

```diff
@@ -0,0 +1,56 @@
+module CC
+  module Analyzer
+    class IssueLocationExistenceValidation < Validation
+      def valid?
+        if location["lines"]
+          valid_lines?
+        elsif location["positions"]
```

Comment:
Oh, wait, you're implicitly returning `nil` if neither of these checks is `true`, which counts as not valid. Right? Nevermind, then.